activate_env.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+# set the environment variables for bmf
+export C_INCLUDE_PATH=${C_INCLUDE_PATH}:$(pwd)/output/bmf/include
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)